docs/src/adoc/index.adoc

```diff
@@ -3848,6 +3848,22 @@ class MyBean {
 }
 ----
 
+Fluent API:
+
+[source,java]
+----
+QualifiedType<MyJson> qualifier = QualifiedType.of(MyJson.class).with(Json.class);
+
+h.createUpdate("insert into myjsons(json) values(:json)")
+    .bindByType("json", new MyJson(), qualifier)
+    .execute();
+
+MyJson result = h.createQuery("select json from myjsons")
+    .mapTo(MyJson.class, Json.class)
```

Comment:
This example is inconsistent: how come we use `QualifiedType` on the `bindByType` call but break it apart for `mapTo`.  Shouldn't they be symmetric?